vendor/github.com/ServiceComb/paas-lager/.travis.yml

```diff
@@ -0,0 +1,36 @@
+language: go
```

Comment:
I don't think file is needed.

Reply:
Kept the file, as dep ensure brings it back